### PR TITLE
December 2023 event agenda

### PR DESCRIPTION
**Pull request by harisood**
Agenda for the website for December event.

### CODE REVIEW

File: docs/events/wg_workshops/2023-12-05-december-meeting/discussion-tre-federation.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# TRE Federation: appetite for a standards working group(s)?
+
+**Chair**: Rob Baxter (DARE UK)
+
+## Prompts
+
+Federating TREs to enable linked-data research is a big topic, with lots of current activity.
+DARE UK have tried, in the spirit of George Box\*, to draw the current threads together into a common architecture.
```
Comment (manics) [inline docs/events/wg_workshops/2023-12-05-december-meeting/discussion-tre-federation.md:8]: Myst supports proper footnotes:
https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#footnotes

**Review by manics — COMMENTED** [review]
Looks fine to me, the title still says `[WIP]` though.

FYI you can also open PRs as "Draft PRs" as well as using `WIP`.

File: docs/events/wg_workshops/2023-12-05-december-meeting/workshop-package-allow-lists.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Package allow lists
+
+**Lead:** Jim Madge (Alan Turing Institute)
```
Comment (Davsarper) [inline docs/events/wg_workshops/2023-12-05-december-meeting/workshop-package-allow-lists.md:3]:
```suggestion
**Lead:** Jim Madge (The Alan Turing Institute)
```

**Review by Davsarper — APPROVED** [review]
One change and you can merge

**Review by manics — APPROVED** [review]

**Pull request merged**